app/api/connectors/BaseDownstreamConnector.scala

```diff
@@ -28,12 +28,11 @@ import scala.concurrent.{ExecutionContext, Future}
 trait BaseDownstreamConnector extends Logging {
   val http: HttpClient
   val appConfig: AppConfig
+  val featureSwitches: FeatureSwitches
```

Comment:
What's the value of adding the feature switch declaration here? I can't see where it gets used, only in each connector module as an implicit. Does it need to be in the interface?

Reply:
Good spot! I will remove (and remove the unnecessary `val`s from each of the connectors).